packages/components/psammead-brand/src/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+import { C_POSTBOX, C_WHITE } from '@bbc/psammead-styles/colours';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-constants/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_HLF,
+  group3ScreenWidthMax,
+} from '@bbc/gel-constants/spacings';
+
+const layoutWrapperWithoutGrid = styled.css`
```

Comment:
You need to change this from `styled.css` to

```JS
import { css } from 'styled-components';

const layoutWrapperWithoutGrid = css`
  ...
`
```

Then everything will work as expected! `styled.css` is not a thing :blush:

Hope that helps